keyboards/3w6/keymaps/manna-harbour_miryoku/config.h

```diff
@@ -0,0 +1,16 @@
+// generated from users/manna-harbour_miryoku/miryoku.org  -*- buffer-read-only: t -*-
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15